src/js/apps/patients/patient/action/action_app.js

```diff
@@ -14,9 +15,11 @@ export default App.extend({
       const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
       const states = currentWorkspace.getStates();
 
+      const defaultInitialState = first(states.filter({ status: 'queued' }));
```

Comment:
Make make this logic part of `states` or `currentWorkspace`, so that you can do:

```js
const defaultInitialState = currentWorkspace.getDefaultState()
```